apps/morph_check

```diff
@@ -193,7 +193,7 @@ if __name__ == '__main__':
     yaml_path = args.config
     SEPARATOR = '=' * 40
 
-    config = {'neuron_checks': [], 'options': []}
+    config = {'neuron_checks': [], 'options': {}}
```

Comment:
I think it can go inside the else statement. I just left it there just in case something unexpected happens. Is there any way to put tests in the apps?